sphinx/source/operators/start_network.rst

```diff
@@ -54,7 +53,6 @@ To add a new node to an existing opening network, other nodes should be started
 
     $ cchost
```

Comment:
Can we have a Debug/Debugging section that says "if you are going to be debugging, pass this flag and make sure the config has X, and be aware of the fact that your security guarantees are null and void", perhaps with a link to doc that exists about oegdb if any (or mention oegdb otherwise).